CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.41.1-2
+* Drop traffic to 127.0.0.1 that isn't originated from the host
```

Comment:
Don;t we still need to add the 'Cached Agent Version' line along with this one?

Reply:
i don't think so since that's not a new change. we cache agent 1.41.1 in init 1.41.1-1 and we still are

Reply:
Good point.